MusicTaste/MusicTaste/MatchingAlgorithm.m

```diff
@@ -81,9 +81,9 @@ - (void) compareSpotifyData:(PFObject *)potentialMatch withData:(NSManagedObject
     for(NSDictionary *match in matchArray){
         [matchesDict addEntriesFromDictionary: match];
     }
-    //Saves matches to Database
     NSMutableArray *matchesArray = [NSMutableArray array];
-    [matchesArray addObject:matchesDict];
+    matchesArray = [matchesDict keysSortedByValueUsingSelector:@selector(compare:)];
```

Comment:
This is where I organize my matches array according to there similarity

Reply:
Usually we use the term "sort" which is more specific than "organize". You could organize something by grouping similar things together, for example. But if you sort them then they will be grouped and *in order*.